CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Yii Framework 2 mongodb extension Change Log
 - Bug #145: Fixed `yii\mongodb\ActiveFixture` fails to find default data file if `collectionName` is specified in array format (klimov-paul)
 - Bug #146: Fixed `yii\mongodb\ActiveRecord` and `yii\mongodb\file\ActiveRecord` looses `_id` custom value on insertion (lxyfirst, klimov-paul)
 - Enh: Added `yii\mongodb\Migration::listCollections()` method (klimov-paul)
+- Enh #147: Added unknown methods `stream_seek` and `stream_tell` to `yii\mongodb\file\StreamWrapper` for `fseek` and `ftell` (AstRonin)
```

Comment:
Need to add brackets: `()` for the PHP function name mentioned: `fseek()` and `ftell()`.